qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1 Send CV.feature

```diff
@@ -0,0 +1,185 @@
+Feature: 3.1 Send CV.feature
+
+Background:
+Given user opened the page with the form to send CV
+
+Scenario Outline: 3.1_01 Different cases entering data in fields "Name", "Email", attach CV, mark checkbox
+When user enters name <Name>
+And user enters email <Email>
+And user <attach_notattach> CV <CV>
+And user <mark_notmark> checkbox <checkbox>
+And user pushes the "Send" button
+Then result <result> should be <error_noerror>
```

Comment:
I think it would be better to write more explicitly, for example:
Then user should see result corresponding <result>

Reply:
Thanks for your comment!
But I 've done some research about it and so I think that we can't write string like this (I mean, like your example), because 1) according to the rules of Gherkin syntax we need to use angle brackets in this case. And 2) also it's no need to write result in terms of user. 


![image](https://user-images.githubusercontent.com/94701044/143900125-495230fb-553b-4751-9fa5-13dc8e06e7d3.png)

Reply:
Okey, I just not able to add angle brackets in a comment here, github eat them for some reason))
Yes I think you don't need to write result only from user, but I meant that all lines (Given, When, Then) should be written in same manner since I heard that somewhere in our course, and maybe I'm wrong and it doesn't matter:)

Reply:
ok, I got it.